src/qt/clientmodel.cpp

```diff
@@ -288,7 +289,7 @@ static void BlockTipChanged(ClientModel* clientmodel, SynchronizationState sync_
     const bool throttle = (sync_state != SynchronizationState::POST_INIT && !fHeader) || sync_state == SynchronizationState::INIT_REINDEX;
     const int64_t now = throttle ? GetTimeMillis() : 0;
     int64_t& nLastUpdateNotification = fHeader ? nLastHeaderTipUpdateNotification : nLastBlockTipUpdateNotification;
-    if (throttle && now < nLastUpdateNotification + MODEL_UPDATE_DELAY) {
+    if (throttle && now < nLastUpdateNotification + MODEL_UPDATE_DELAY.count()) {
```

Comment:
nit:
```suggestion
    if (throttle && now < nLastUpdateNotification + count_milliseconds(MODEL_UPDATE_DELAY)) {
```

We could clarify the unit.

Reply:
I think this is a fine clarification if others think it makes this code more readable, additionally there is this line of guidance: 

https://github.com/bitcoin-core/gui/blob/6182e5086fc82b49a5f0d2abdab4bc7e2a4686ef/src/util/time.h#L22-L24

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/517#issuecomment-1008325109).